Tests/Common/DIGraphTests.swift

```diff
@@ -3,8 +3,9 @@ import Foundation
 import SharedTests
 import XCTest
 
-class DIGraphTests: IntegrationTest {
+class DIGraphTests: UnitTest {
```

Comment:
Updated the test to inherit from `UnitTest` instead because `Common` module has no longer access to any `IntegrationTest` and we can neither really initialize the SDK or any module in it. However, the test should work fine using `UnitTest` too.

Reply:
This might be beyond the scope of this PR, but I wonder...

I see this test class is in `Common` module. This PR includes modifications to digraph in other modules such as `Sources/MessagingPush/autogenerated/AutoDependencyInjection.generated.swift`. Because of that, I was expecting this PR to also be modifying a test class such as: `Tests/MessagingPush/DIGraphTests.swift`, but it looks like that file doesn't exist today. Even in `main`. 

So, I don't think that the file `Sources/MessagingPush/autogenerated/AutoDependencyInjection.generated.swift` that got modified in this file is getting tested?

Reply:
Sorry, couldn't get your point completely. I can see we have [`DIGraphTests` in `MessagingPush`](https://github.com/customerio/customerio-ios/blob/main-replica-for-cdp/Tests/MessagingPush/DITests.swift). Isn't it enough to test changes for `Sources/MessagingPush/autogenerated/AutoDependencyInjection.generated.swift`?

Also, after reading your comment, I do realize it was missing for `DataPipeline` so [added them](https://github.com/customerio/customerio-ios/pull/501/commits/e3564ea0112ea312ab4226559f684b7c2c4d3adb).